README

```diff
@@ -7,15 +7,24 @@ This package and the code it generates requires at least Go 1.4.
 This software implements Go bindings for protocol buffers.  For
 information about protocol buffers themselves, see
 	https://developers.google.com/protocol-buffers/
-To use this software, you must first install the standard C++
-implementation of protocol buffers from
+
+## Installation ##
+
+To use this software, you must:
+- Install the standard C++ implementation of protocol buffers from
 	https://developers.google.com/protocol-buffers/
-And of course you must also install the Go compiler and tools from
+- Of course, install the Go compiler and tools from
 	https://golang.org/
-See
+  See
 	https://golang.org/doc/install
-for details or, if you are using gccgo, follow the instructions at
+  for details or, if you are using gccgo, follow the instructions at
 	https://golang.org/doc/install/gccgo
+- Grab the code from the repository and install the proto package. The simplest
```

Comment:
Done. Moved it below. I agree, it is easier to read when "The simplest ..." sentence is on its own line.